### ISSUE TITLE
[Feature] Add save/state sync with server

### ISSUE THREAD

**Issue by gantoine** [rommapp/playnite-plugin#6]

**Comment by ctv99** [comment]
I Second this!

**Comment by Oxika95** [comment]
I am legitimately baffled this isn't already a part of the plugin. Isn't the cloud-based nature of romM the primary reason to use a self-hosted game library? I just installed playnite and the plugin only to realize I can't play my saves without manually downloading them first which seems to defeat the purpose? or is the plugin only meant to be used for library migration?

**Comment by gantoine** [comment]
Yes! My C# skills are at a 1/10, so hoping I can rope Claude into helping with this one (or better yet, find a c# dev willing to help).

**Comment by Danjameswalker** [comment]
With the syncing updates to Romm is this on the cards now?

**Comment by gantoine** [comment]
https://github.com/rommapp/playnite-plugin/pull/131
